hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/procedure/TestCompactionProcedure.scala

```diff
@@ -360,7 +360,7 @@ class TestCompactionProcedure extends HoodieSparkProcedureTestBase {
         val conf = new Configuration
         val metaClient = HoodieTestUtils.createMetaClient(new HadoopStorageConfiguration(conf), basePath)
 
-        assert(0 == metaClient.getActiveTimeline.getCompletedReplaceTimeline.getInstants.size())
+        assert(metaClient.getActiveTimeline.getCompletedReplaceTimeline.getInstants.size() == 0)
```

Comment:
Swapping the RHS and LHS here as scalatest's assert will use the RHS as "expected", while the LHS is "actual". 

It's an OCD thing, but test hints will look correct when there are errors.